recipes/kangaru/all/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.1)
 project(test_package CXX)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
+find_package(kangaru REQUIRED CONFIG)
+
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} kangaru::kangaru) # TODO: remove kangaru:: namespace when fixed in cpp_info of conanfile.py
```

Comment:
do we already have an issue for that?

Reply:
I don't think so. I could create one, a lot of recipes in cci requires this feature.

Reply:
yes please, feel free to go ahead and create one. if you happen to create one eventually, please leave link here as well.